### ISSUE TITLE
Passing multiple parameters with $emit

### ISSUE THREAD

**Issue by evergreenwebsystems** [calebporzio/livewire#110]
**Describe the bug**
Prior to the latest release I used to be able to pass multiple parameters with the `$emit` event. However, now I can only access the first parameter in my component.

For example I had this in my blade template:
```
wire:click="$emit('showModal', 'projects.edit', '2')"
```

Which I could then access in my Livewire component like this:
```
    protected $listeners = ['showModal' => 'open'];

    public function open($type, $id)
    {
        $this->isOpen = true;
        $this->type = $type;
        $this->id = $id;
    }
```

Now, both `$type` and `$id` are received as 'projects.edit'. So, `$id` is missing and `$type` is received twice.

Am I doing something wrong or has the code changed with respect to this behaviour?

**Comment by evergreenwebsystems** [comment]
I can confirm that issue is only related to the current release, version 0.1.2. I downgraded to version 0.1.1 and ran the same code and encountered no problems.

**Comment by calebporzio** [comment]
Shoot, thanks for catching that. Fixed in v0.1.3

**Comment by evergreenwebsystems** [comment]
I have just updated Livewire to v0.3.10 and this issue seems to have re-appeared.

```
wire:click="$emit('showModal', 'projects.edit', '2')"
```

```
    public function open($type, $id)
    {
        $this->isOpen = true;
        $this->type = $type;
        $this->id = $id;
    }
```

I only receive ```$type```, the second parameter ```$id``` has disappeared entirely.

Has something changed?
